src/bridge/SequencerInbox.sol

```diff
@@ -56,7 +54,11 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
 
     IOwnable public rollup;
```

Comment:
why was rollup kept mutable? in anticipation for BOLD update or sthg?

Reply:
Yeh, we may ship a new rollup and it would be nice not to have to update the seq inbox. As it happens the rollup address isnt used on the happy path, so making it mutable doesnt affect the gas cost of batch posting